EPRN.PRNS.API/Controllers/PRNController.cs

```diff
@@ -76,7 +76,7 @@ public async Task<IActionResult> GetTonnage(int id)
         public async Task<IActionResult> SaveCheckYourAnswersState(
             int id)
         {
-            await _prnService.SaveCheckYourAnswers(id);
+            await _prnService.SaveCheckYourAnswers(id, "Update");
```

Comment:
Firstly, why pass this in from the controller? Set it at the service level - that's what it's for. Secondly, if we are going to be updating status' without a reason, then we _might_ want to consider making reason not a required field. You might want to discuss this.

Reply:
Done.